setup.cfg

```diff
@@ -0,0 +1,3 @@
+[metadata]
+license_files =
+   LICENSE.txt
```

Comment:
Could put these on the same line I think (or maybe GitHub is rendering this weird for me)?

Reply:
I think this is a list of one item. There was a second item from the docs you linked to that I removed.

Reply:
It works fine if you have only one license and include it on the same line. Here's [an example]( https://github.com/dask/dask/blob/fbc373741264244102efc25d53893d0d83e669a6/setup.cfg#L48-L49 ).